docs/howto/features/cryptnono.md

```diff
@@ -0,0 +1,176 @@
+(howto:features:cryptnono)=
+# Enable stronger anti-crypto abuse features for a hub
+
+For JupyterHubs and BinderHubs that are broadly open to the public, cryptomining
+attacks are the most common security threat. They take up resources and rack up
+cloud costs. While most hubs can get away with gating access to them via some
+kind of login restriction, for a subset of hubs this is not ideal (for equity and
+access reasons).
+
+For those cases, we enable a stronger deployment of [cryptnono](https://github.com/cryptnono/cryptnono).
+The cryptnono project (in use on mybinder.org as well) helps detect and kill cryptominers via
+various 'Detectors'.
+
+## Detectors
+
+Cryptnono currently has two primary detectors:
+
+1. A detector for the [monero](https://www.getmonero.org/) cryptocurrency, that is based on
+   official guidance [from the monero project](https://blog.px.dev/detect-monero-miners/) on
+   how to detect it. This is fairly safe and has a very low false positive rate, and requires
+   no configuration. So by default, **this detector is enabled on all hubs**.
+
+2. A detector (`execWhacker`) based on heuristics, where the full commandline used to execute a process (
+   regardless of how it is started) is used to detect if a process is likely crypto mining,
+   and if so, immediately kill it. This relies on a tweakable config of banned strings to
+   look for in the commandline of a process, and is [constantly being tweaked](https://github.com/jupyterhub/mybinder.org-deploy/security/advisories/GHSA-j42g-x8qw-jjfh).
+   Since making the list of banned strings public would make ban evasion easy, the list of
+   strings (and the method used to generate them) is encrypted. You can read more details
+   about the specific method used by looking in the encrypted code file (`deployer/commands/generate/cryptnono_config/encrypted_secret_blocklist.py`)
+   in this repository.
```

Comment:
```suggestion
   in the `infrastructure` repository.
```

I don't think it's clear when you're on https://infrastructure.2i2c.org which repository 'this' one is.